vue/src/Tag/TagDateInput.vue

```diff
@@ -166,7 +166,7 @@ export default defineComponent({
       .timepicker({ timeFormat: 'H:i:s' })
       // timepicker triggers a jquery event, not a addEventListener event, so vue doesn't catch
       // it
-      .on('change', (event) => {
+      .on('change', (event: ChangeEvent) => {
```

Comment:
`./console vue:build TagManager` was failing and had to update as below to make it work